data/monarchs.json

```diff
@@ -1,10 +1,10 @@
 [
-{"name":"Elizabeth","start":1565,"end":1603,"index":0},
+{"name":"Elizabeth I","start":1565,"end":1603,"index":0},
 {"name":"James I","start":1603,"end":1625,"index":1},
 {"name":"Charles I","start":1625,"end":1649,"index":2},
 {"name":"Cromwell","start":1649,"end":1660,"commonwealth":true,"index":3},
 {"name":"Charles II","start":1660,"end":1685,"index":4},
-{"name":"James II","start":1685,"end":1689,"index":5},
+{"name":"James II","start":1685,"end":1688,"index":5},
 {"name":"W&M","start":1689,"end":1702,"index":6},
```

Comment:
https://www.hrp.org.uk/kensington-palace/history-and-stories/william-iii-and-mary-ii/#gs.e3lczb says 

William III and Mary II were England’s first and only joint sovereigns, with Mary sharing equal status and power. William and Mary came to the throne after the "Glorious Revolution" of 1688 when Mary’s father, James II, was deposed for trying to enforce Catholic tolerance in England. The King and Queen ruled jointly from **1689** until Mary’s death aged 32 in **1694**.

Is 1702 correct?

Reply:
While William & Mary's joint rule ended in 1694, William's solo reign continued until his death in 1702, when he was thrown from a horse and broke his collar bone.  This is noted in the proposed SOURCES.md entry as follows:

>#### William & Mary's Reign
>The entry "W&M" represents the joint reign of William III and Mary II. While the dataset shows their reign as 1689-1702, the official Web site of the British royal family indicates that Mary II's reign ended in 1694, though William III continued to rule until 1702.